subsys/bluetooth/controller/ll_sw/nordic/hal/nrf5/radio/radio.c

```diff
@@ -116,6 +116,10 @@ void radio_reset(void)
 			    & RADIO_POWER_POWER_Msk);
 
 	hal_radio_reset();
+
+#if !defined(CONFIG_BT_CTLR_TIFS_HW)
+	hal_radio_sw_switch_ppi_group_setup();
```

Comment:
This is, actually, the only functional change introduced in the code:
The group configuration (i.e. the inclusion of PPI channels in the two groups) is moved to radio_reset() from radio_tmr_start().

This makes more sense because the group configuration shall be done before calling sw_switch().

Note that at the beginning of radio event the sequence of calls is:

radio_reset() - sw_switch() - radio_tmr_start()